test/scripts/filters/meta_generator.js

```diff
@@ -11,7 +11,7 @@ describe('Meta Generator', () => {
     const result = metaGenerator(content);
 
     const $ = cheerio.load(result);
-    $('meta[name="generator"]').length.should.eql(1);
+    $('meta[name="generator"]').should.to.have.lengthOf(1);
```

Comment:
```suggestion
    $('meta[name="generator"]').should.have.lengthOf(1);
```

Not only `should.to.be` should be replaced with `should.be` but also `should.to.have`  should be replaced with  `should.have`.